raisinbread/README.md

```diff
@@ -29,6 +29,16 @@ that is not the case, replace all `mysql` commands below by the necessary values
 `mysql -u user -p -h host database_name`*
 
 ##### Sourcing SQL
+The script `tools/raisinbread_refresh.php` includes functionality that drops all existing tables in the
```

Comment:
I'm okai with this addition however I don't think it belongs under the `Sourcing SQL` header. This script does so much more than just sourcing the SQL. It actually replaces pretty much this entire readme.

I think we should instead divide the `Installing RB` header into 2 sections. `automated` and `manual` instructions and make it clear to users that you should pick or the other to install it.

Reply:
I can make this change to the formatting. For developers who are first sourcing RB though, they also need to make the steps in the following section "Configuring", which isn't done by this script. I'll try to make this clear.